coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "line": 100.0
+    "line": 98.27
   }
 }
```

Comment:
We should try not to commit these -- when you see them, just git stash these and then ignore them